core/src/main/java/org/apache/calcite/sql/type/OperandTypes.java

```diff
@@ -443,8 +443,9 @@ private boolean hasFractionalPart(BigDecimal bd) {
       family(SqlTypeFamily.STRING, SqlTypeFamily.INTEGER);
 
   public static final SqlSingleOperandTypeChecker STRING_INTEGER_OPTIONAL_INTEGER =
-      family(ImmutableList.of(SqlTypeFamily.STRING, SqlTypeFamily.INTEGER, SqlTypeFamily.INTEGER),
```

Comment:
Why are we breaking here not at the higher level statement, the projects line length is 120.

Reply:
We have a policy to not have two open parentheses on the same line that are not matched by close parentheses. Because otherwise it's not obvious whether the next line should increase indent by 1 or 2. And it's easier to see where a function call begins and ends.

Occasionally, as in this case, it requires an extra line break. But I think you'll agree that the result is quite readable.

Reply:
Could you please point me to the policy so I can be complainant in future pull requets.